### DISCUSSION TITLE
select client by IP range

### DISCUSSION THREAD

**Discussion by GinTonic99** [gerbera/gerbera#2700] [Q&A] [answered]
can I select a set of clients using an IP range in the client section of the config file? I have tried 

`<client IP="10.0.0.0/24" flags=... >
</client>
`

but this does not work. Is there a syntax to select an IP range that is supposed to work?
thanks

**Comment by KarlStraussberger** [comment, marked as answer]
It is not supported. The goal is to handle weirdness of particular devices. If you have more than one, `userAgent` is the better choice. It also works with dynamic IPs.

**Reply by GinTonic99** [reply]
Thank you for the fast response. My idea was to activate transcoding for all clients connected via VPN (and thus having an IP in a specific range).

**Reply by whyman** [reply]
That is a good idea
